index.html

```diff
@@ -0,0 +1,1305 @@
+            alt="CUL_StephenMalkmus_Banner" title="CUL_StephenMalkmus_Banner" width="466" height="311"
+            class="mapping-tablet w-100 img-fluid"
+            src="https://d.newsweek.com/en/full/1571094/cul-stephenmalkmus-banner.jpg?w=466&amp;h=311&amp;l=47&amp;t=45&amp;q=90&amp;f=4b0a45aff01bf29a88a38346cc2de7c0">
+          <div class="card-body pl-0 ">
+            <div class="card-title d-flex">
+              <button class=" b-cover cover">Cover</button>
+              <button class="ml-3 bg-white cover culture">Culture</button>
+            </div>
+            <h4 class="w-100">OK, Millennial: Boomers Are the Greatest Generation in History</h4>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-3 col-md-3 col-sm-12 col-xm-12 w-100">
+        <div class="card mt-5 w-100 ">
+          <img
+            data-src="https://d.newsweek.com/en/full/1571795/fea-okmillennials-banner.jpg?w=466&amp;h=311&amp;q=90&amp;f=22793e3731bb8612bf9e34795327f3e1"
+            alt="FEA_OkMillennials_Banner" title="FEA_OkMillennials_Banner" height="311"
+            class="mapping-tablet w-100 img-fluid"
+            src="https://d.newsweek.com/en/full/1571795/fea-okmillennials-banner.jpg?w=466&amp;h=311&amp;q=90&amp;f=22793e3731bb8612bf9e34795327f3e1">
+          <div class="card-body pl-0 ">
+            <div class="card-title d-flex">
+              <button class=" b-cover cover">Cover</button>
+              <button class="ml-3 bg-white cover culture">Culture</button>
+            </div>
+            <h4 class="w-100">OK, Millennial: Boomers Are the Greatest Generation in History</h4>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-3 col-md-3 col-sm-12 col-xm-12 w-100">
+        <div class="card mt-5 w-100 ">
+          <img
+            data-src="https://d.newsweek.com/en/full/1571094/cul-stephenmalkmus-banner.jpg?w=466&amp;h=311&amp;l=47&amp;t=45&amp;q=90&amp;f=4b0a45aff01bf29a88a38346cc2de7c0"
+            alt="CUL_StephenMalkmus_Banner" title="CUL_StephenMalkmus_Banner" width="466" height="311"
+            class="mapping-tablet w-100 img-fluid"
+            src="https://d.newsweek.com/en/full/1571094/cul-stephenmalkmus-banner.jpg?w=466&amp;h=311&amp;l=47&amp;t=45&amp;q=90&amp;f=4b0a45aff01bf29a88a38346cc2de7c0">
+          <div class="card-body pl-0 ">
+            <div class="card-title d-flex">
+              <button class=" b-cover cover">Cover</button>
+              <button class="ml-3 bg-white cover culture">Culture</button>
+            </div>
+            <h4 class="w-100">OK, Millennial: Boomers Are the Greatest Generation in History</h4>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="row  pl-lg-4 pr-lg-4 bg-white pt-sm-3">
+      <div class="col-12 mt-4">
+        <div class="d-flex magazine">
+          <h5 class="m-auto d-flex pb-2"><span class="icon-n mr-3"></span>EDITOR'S PICK</h5>
+        </div>
+      </div>
+    </div>
```

Comment:
The article images in this section are wider due to the general width of your page. Please make sure that when you fix the body, this section reflects those changes, too.

Make sure you include the categories' badges inside the images.